TOOLS/dylib-unhell.py

```diff
@@ -80,15 +80,19 @@ def process_libraries(libs_dict, binary, processed = []):
             if p in libs_dict[src]:
                 install_name_tool_change(p, lib_name(p), dst)
 
-    process_libraries(libs_dict, binary, ls)
+    return ls
 
 def main():
     binary = os.path.abspath(sys.argv[1])
     if not os.path.exists(lib_path(binary)):
         os.makedirs(lib_path(binary))
     libs = libraries(binary)
-    print(libs)
-    process_libraries(libs, binary)
+
+    libs_processed = process_libraries(libs, binary)
+    while True:
```

Comment:
`while libs_processed is not None`?

Reply:
indeed, like that it doesn't make much sense. i had a different approach and condition there before that didn't work properly and later saw it could be done a lot easier.

pushed the change.